SplunkRumWorkspace/SplunkRum/SplunkRumTests/UtilsTests.swift

```diff
@@ -196,12 +196,12 @@ class UtilsTests: XCTestCase {
         _ = re.export(spans: fiftySpans)
         _ = re.export(spans: fiftySpans)
         _ = re.export(spans: fiftySpans)
-        XCTAssertEqual(0, localSpans.count)
+        XCTAssertEqual(150, localSpans.count)
 
         // now reconnected; next export of 50 should send total of 150 (last 100 retried)
         te.exportSucceeds = true
         _ = re.export(spans: fiftySpans)
-        XCTAssertEqual(150, localSpans.count)
+        XCTAssertEqual(250, localSpans.count)
```

Comment:
Perhaps we need additional tests besides this?

Reply:
@seemk added additional tests.

Reply:
Can we verify that serialized attributes are all loaded correctly? I.e. when constructing spans from disk

Reply:
Add test case to check that attributes are loaded properly.